cdp-core.md

```diff
@@ -113,7 +113,7 @@ Vat CDP State
         <cat>
           <cat-ward> .Map </cat-ward>
           <cat-ilks> .Map </cat-ilks>
-          <cat-live> 0    </cat-live>
+          <cat-live> 1    </cat-live>
```

Comment:
Can we use `true` instead?

Reply:
Let's make that into a separate PR.

Reply:
I made a card for it.

https://github.com/runtimeverification/mkr-mcd-spec/projects/1#card-26514903